api/envoy/extensions/filters/http/mutation/v3alpha/processing_mode.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.mutation.v3alpha;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.mutation.v3alpha";
+option java_outer_classname = "ProcessingModeProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Mutation Service]
+// Mutation Service Processing Mode
+// [#extension: envoy.filters.http.mutation]
+
+// This configuration describes how a mutation filter processes the HTTP
+// request or response -- specifically, whether it is done synchronously
+// or asynchronously.
+
+message ProcessingMode {
+  enum Mode {
+    // In the default "sync" mode, a message is sent to the filter server
+    // as soon as decodeHeaders is called, and the filter waits for a response
+    // from the server before continuing.
+    SYNC = 0;
```

Comment:
Per my other comment we will need per-op timeouts. It might be fine to have a timeout for each ACK/mutation, or we might end up needing per-op timeouts? Can you flesh that out?